integration-tests/scripts/devnet-hardhat.sh

```diff
@@ -57,5 +61,5 @@ fi
 echo "Starting hardhat..."
 docker run --net container:devnet_local -d ethereumoptimism/hardhat
 echo "Starting L1<>L2 tests"
-npx hardhat test test/postman.test.ts --starknet-network devnet --network localhost
-
+# npx hardhat test test/postman.test.ts --starknet-network devnet --network localhost
+yarn test
```

Comment:
Let's keep this script only starting the environment, and we move/group the test parts in `scripts/test.sh`.